src/styles/_catalog.scss

```diff
@@ -14,9 +14,18 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
-  }
+    transition: all 300ms;
+
```

Comment:
maybe better to define exactly which properties will be animated, not all?

Reply:
Better to define which exact properties will be animated, because if you not define it, it will be applied to every property that can be animated